addon/components/nrg-modal/component.js

```diff
@@ -21,6 +21,7 @@ export default Component.extend({
   basic: false,
   sidebar: false,
   lightbox: false,
+  scrolling: true,
```

Comment:
Should this be a `reads` value of `!this.renderInPlace && !this.masterDetail && !this.sidebar && !this.lightbox` instead?